src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java

```diff
@@ -0,0 +1,92 @@
+package com.example.bookstoreapp.services.impl;
+
+import com.example.bookstoreapp.entities.MediaEntity;
+import com.example.bookstoreapp.models.Media;
+import com.example.bookstoreapp.repositories.MediaEntityRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MediaServiceImplTest {
```

Comment:
Your class file name is MediaServiceImpl (src/test/java/com/example/bookstoreapp/services/impl/MediaServiceImpl.java) but the class name defined at line 17 here I see MediaServiceImplTest.

I think both should named as MediaServiceImplTest.

Reply:
ok